checks/normandy/recipes_signatures.py

```diff
@@ -0,0 +1,47 @@
+"""
+Signatures should be valid for each published recipe.
+
+The list of failing recipes is returned.
+"""
+import json
+import logging
+
+from autograph_utils import MemoryCache, SignatureVerifier, decode_mozilla_hash
+
+from poucave.typings import CheckResult
+from poucave.utils import ClientSession, fetch_json
+
+
+RECIPES_URL = "{server}/buckets/main/collections/{collection}/records"
+
+
+logger = logging.getLogger(__name__)
+
+
+async def validate_signature(verifier, recipe):
+    signature_payload = recipe["signature"]
+    x5u = signature_payload["x5u"]
+    signature = signature_payload["signature"]
+    attributes = recipe["recipe"]
+    data = json.dumps(attributes, sort_keys=True, separators=(",", ":")).encode("utf8")
+    return await verifier.verify(data, signature, x5u)
+
+
+async def run(server: str, collection: str, root_hash: str) -> CheckResult:
```

Comment:
It would be nice to have a doc string explaining the expected values for these parameters, or to include them in the module doc string.